projects/vc-api-implementation.md

```diff
@@ -0,0 +1,240 @@
+
+*Elia and Energy Web are open to adopting governance practices as suggested by OWF.* 
+
+# Links to Documented Governance Practices
+_Include links to any documented governance practices._
+
+Contribution guidelines: https://github.com/energywebfoundation/ssi/blob/develop/contributing.md  
+
+# Preferred Maturity Level
+_Specify the stage that you want this project to enter. See [project lifecycle](https://openwallet-foundation.github.io/tac/governance/project-lifecycle/#stages) for information on the different stages._
+
+The requested maturity level is “Labs”.  
+
+Though the project team believes that the “Labs” stage is appropriate, the following points highlight early signs of project usefulness and maturity: 
+
+- This implementation is being utilized by BloxMove proof of concepts to accomplish the next real-world Mobility use-cases. 
+- Elia has integrated its wallet with the project, enabling users to leverage it for various energy use cases. 
+- The project is accompanied by comprehensive technical documentation and tutorials, aimed at facilitating adoption. 
+
+# Communication Channels
+_If already exists, include a list of project's official communication channels (slack, irc, mailing lists)._
+
+GitHub Issues in the project's repository are enabled and publicly available. The project does not currently have any additional communication channels. 
+
+# Project Website
+_If exists, include a link to project's website._
+
+# Social Media
+_If exists, include links to social media accounts._
+
+Elia made a short video to introduce the concept of SSI to newcomers: https://eliagroup.sharepoint.com/:v:/s/InnovationTeam/EUfrphynLWlLsMhJ8mMkjboBxIHTyyoy3RqGalcdFGqn-g?e=JcYOLU  
+
+# Financial Sponsorship
+_Include any existing financial sponsorship._
+
+Elia has funded the development of the project and is currently funding maintenance and minor feature development, in the context of initiatives which are ongoing through 2023. Furthermore, Elia has made funding available for any changes required for the project to be donated to the OWF. 
+
+Beyond the context of the ongoing initiatives and donation of the project to OWF, Elia may provide future funding. Questions which Elia plans use to evaluate provision of future funding include: 
+- Would the funding enable changes which better align the implementation to the VC API specification? 
+- Would the funding enable changes which improve support of the implement for Elia’s use cases? 
+
+# Infrastructure
+_Include any infrastructure needs or requests._
+
+The following is requested as repository infrastructure:
+
+| Infrastructure Technology | Purpose  |
+|---------------------------|----------|
+| GitHub Actions | Continuous integration and release workflows |
+
+The following is requested as release artifact infrastructure: 
+
+| Infrastructure Technology | Purpose |
+|---------------------------|----------|
+| *no technology preference* | Public container image repository |
```

Comment:
Would ghcr.io work?

Reply:
Yes, ghcr.io would work 👍 . Added to proposal as example here: https://github.com/openwallet-foundation/project-proposals/pull/12/commits/59807085ae9c6203bca47603dc603740279e9dfb